src/site/xdoc/server/config-elasticsearch.xml

```diff
@@ -186,6 +190,66 @@
 
     </section>
 
+    <section name="SSL Trusting Configuration">
+
+        <p>
+            By default James will use the system TrustStore to validate https server certificates, if the certificate on
+            ES side is already in the system TrustStore, you can leave the sslValidationStrategy property empty or set it to default.
+        </p>
+
+        <dl>
+            <dt><strong>elasticsearch.hostScheme.https.sslValidationStrategy</strong></dt>
+            <dd>
+              Optional. Accept only <strong>default</strong>, <strong>ignore</strong>, <strong>override</strong>. Default is <strong>default</strong>
+            </dd>
+            <dd>
+                default: Use the default SSL TrustStore of the system.
+                ignore: Ignore SSL Validation check (not recommended).
+                override: Override the SSL Context to use a custome TrustStore containing ES server's certificate.
+            </dd>
+        </dl>
+
+        <p>
+            In some cases, you want to secure ES to protect it from unauthorized requests,
+            assuming with the ES is using <strong>https</strong> with a self signed certificate.
```

Comment:
we need a little more explanation here: I'm not sure having nginx in front of ES is a so common practice. So can you explain a little our setup?

Reply:
I prefer to put those steps into how-to documentation. do you agree?

Reply:
I don't speak about the steps of starting ES with nginx base authentication, I'm just speaking of explaining that we don't use native (non-free?) ES authentication, but instead nginx (or reverse-proxy) based authentication.

Reply:
I’m not sure but I think it can work well with a paid ES + xpack. Because I used nothing but just options provided by the RestHighLevelClient API. I mean from the client side, the configuration looks the same regardless what kind of ES. So why do you want to explain it, Im sorry I dont get it

Reply:
in this case it's ok, thank you